tests/PHPUnit/System/BackwardsCompatibility1XTest.php

```diff
@@ -118,6 +118,7 @@ public function getApiForTesting()
 
             // those reports generate a different segment as a different raw value was stored that time
             'DevicesDetection.getOsVersions',
+            'DevicesDetection.getBrowserFamilies',
```

Comment:
shouldn't `DevicesDetection.getBrowsers` be in the list as well? Also are the expected files for those methods still needed then? Guess those methods are skipped for testing 🤔

Reply:
Added it now. The failing test for getBrowsers was initially not shown on travis only after getBrowserFamilies test was fixed. The test is not skipped but instead have a separate fixture so the files be still needed I suppose.